src/main/java/Entities/CurrentUser.java

```diff
@@ -0,0 +1,26 @@
+package Entities;
+import Entities.User;
```

Comment:
don't know if you have to import Entities.User if this class is already in the Entities package.